runtimes/eden/src/xcm_config.rs

```diff
@@ -206,7 +206,7 @@ impl orml_xtokens::Config for Runtime {
 	type AccountIdToMultiLocation = AccountIdToMultiLocation;
 	type SelfLocation = SelfLocation;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type Weigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
+	type Weigher = WeightInfoBounds<crate::weights::NodleXcmWeight<RuntimeCall>, RuntimeCall, MaxInstructions>;
```

Comment:
I suspect this weigher to not be used as I did my tests and the weigher from the pallet_xcm was making the transaction to fail

Reply:
Yes you can change pallet_xcm to use `FixedWeightBounds` or xTokens to use `WeightInfoBounds`. The point is both of them seem to need to use the same weigher. In this case, the benchmarked weigher is the xcm one and that's why I changed xTokens.